### PR TITLE
Contact screen

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: src/shared/components/Input/styles.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  color: $lightGrey;
+}
+
+.input__mandatory-mark {
+  color: $red;
+}
+
+input[value]:not([value='']) + .input__placeholder {
+  display: none;
+}
```
Comment (mi-chal) [inline src/shared/components/Input/styles.scss]: I think you don't need it anymore

File: src/shared/components/Logo/styles.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
+@import 'shared/styles/colors.scss';
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;300&display=swap');
```
Comment (mi-chal) [inline src/shared/components/Logo/styles.scss:3]: Remove

File: src/shared/components/Navigation/styles.scss [hunk 3]
```diff
@@ -1,34 +1,29 @@
 @import 'shared/styles/colors.scss';
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@1,100&display=swap');
 
 .navigation {
   display: flex;
-  background-color: $white;
-  height: 80px;
+  height: 30px;
```
Comment (mi-chal) [inline src/shared/components/Navigation/styles.scss:5]: No need to set fixed height

**Review by mi-chal — APPROVED** [review]

File: src/shared/components/Input/styles.scss (hunk 1 above)
Comment (dwantula) [reply src/shared/components/Input/styles.scss]: Fixed

File: src/shared/components/Logo/styles.scss (hunk 2 above)
Comment (dwantula) [reply src/shared/components/Logo/styles.scss:3]: Done

File: src/shared/components/Navigation/styles.scss (hunk 3 above)
Comment (dwantula) [reply src/shared/components/Navigation/styles.scss:5]: Done

**Pull request merged**